server/src/catalog.rs

```diff
@@ -69,33 +69,33 @@ impl ManifestFile for manifest::File {
     }
 }
 
+fn get_file_bounds(file: &manifest::File) -> (DateTime<Utc>, DateTime<Utc>) {
+    match file
+        .columns()
+        .iter()
+        .find(|col| col.name == "p_timestamp")
+        .unwrap()
+        .stats
+        .clone()
+        .unwrap()
+    {
+        column::TypedStatistics::Int(stats) => (
+            NaiveDateTime::from_timestamp_millis(stats.min)
+                .unwrap()
+                .and_utc(),
+            NaiveDateTime::from_timestamp_millis(stats.min)
```

Comment:
@gurjotkaur20 do you intend to get the min and max of p_timestamp from manifest.json file? I can see stats.min and stats.min. Please check.

Reply:
yes, that's why I have tried to use the same function to find the min(the first event timestamp) from the manifest.json file.